### PR TITLE
Finish query() function in FindMeetingQuery, wrote JUnit tests, fixed bugs in datastore for changing nicknames and sentiment scores

### PR DESCRIPTION
**Pull request by juliasliu**

### CODE REVIEW

**Review by wenyi-guo — APPROVED** [review]
Looks great! You finished everything

**Comment by ajs11893** [comment]
The instructions say cd into the `projects` folder but it should say cd into `step/walkthroughs/week-5-tdd/project` or `project`

File: README.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+
+To run the app locally, `cd` into the `portfolio` directory and `mvn package appengine:run`.
+
+## Week 5 Project: TDD (Test Driven Development)
+
+Code for the query() function is located in `step/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java`.
+
+JUnit tests are located under the `step/walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java`.
+
+To run the tests, `cd` into the `projects` directory and `mvn test` to run the JUnit tests.
```
Comment (ajs11893) [inline README.md:18]: I think it should `project` not `projects`

**Review by ajs11893 — APPROVED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (juliasliu) [reply README.md:18]: Yeah you're right, I'll change that and then try to merge.

**Pull request merged**